lib/handlers/route-api.js

```diff
@@ -47,7 +47,8 @@ export class RouteAPI {
             .get('/z/:id', _.bind(this.storedStateHandler, this))
             .get('/z/:id/code/:session', _.bind(this.storedCodeHandler, this))
             .get('/resetlayout/:id', _.bind(this.storedStateHandlerResetLayout, this))
-            .get('/clientstate/:clientstatebase64', _.bind(this.unstoredStateHandler, this));
+            .get('/clientstate/:clientstatebase64', _.bind(this.unstoredStateHandler, this))
+            .get('/unstored', _.bind(this.unstoredCodeHandler, this));
```

Comment:
What is the "unstored" here? I can't think what this name would be. Is this like a super simple API call to make a plain old side-by-side compiler view? If so...I can't really think of a better name either lol but maybe someone else watching can do. (`fromsimplelayout` too long, something super short like `l` not helpful...er... sorry. this comment is unhelpful too haha, just y'know APIs are long-lived and all that...)

Can you mention in the API doc, unless you want this to be hidden? Maybe an example showing how it could be used to solve #2670 (I am being dumb and can't see how...)

Reply:
`fromsimplelayout` seems reasonable to me. It's used via query parameters like the issue calls for. (and how it's implemented in Zulip)
So `/fromsimplelayout?lang=cpp&compiler=g101&compiler_flags=-O3&code=int+main()+{+return+1;};`
It then builds a simple session with the code and the compiler